packages/website/src/pages/rides.js

```diff
@@ -25,11 +25,11 @@ class RidesPage extends Component {
           <p>Details of weekend rides are usually posted to <a href='https://www.facebook.com/PeckhamCC'>Facebook</a> and <a href='https://twitter.com/peckhamcc'>Twitter</a> towards the end of the week.</p>
           <p>First time riders are more than welcome but if you are thinking of joining us for the first time, please get in touch first so we know to look out for you!</p>
           <h3>Social Spin</h3>
-          <p>There is a club social ride on most Saturdays throughout the year and is the ideal introduction to riding with Peckham CC.</p>
+          <p>There is a club social ride on every Saturday throughout the year and is the ideal introduction to riding with Peckham CC.</p>
           <p>It's a non-drop ride, meaning we regroup at the tops of climbs and everyone rides back together.</p>
-          <p>The '<Link to='/routes'>short loop</Link>' is about 60km long and takes about three hours.  At the 30km point we usually decide whether to go further, sometimes splitting the group if people have to be back early.</p>
+          <p>The '<Link to='/routes'>short loop</Link>' is about 60km long and takes about three hours. There's always multiple groups at the library, and there's generally a group of people interested in riding further and/or slightly faster.</p>
           <h3>Ladies Ride</h3>
-          <p>On the third Sunday of every month there is a women-only ride aimed at riders of all abilities.  The route varies month to month but is about 50km in length and the group leaves the library at 9am.</p>
+          <p>During summer, once a month there is a women-only ride aimed at riders of all abilities. The route varies month to month and is designed to offer women and non-male identifying people a safe and inclusive space to stretch themselves. The pace is inclusive and the distance is generally slightly further than the social ride.</p>
```

Comment:
I don't think this is true, summer official starts at the solstice (e.g. end of June) but the chat in the room is to start the rides up again at the beginning of March.  Maybe say something like every month from spring to autumn?

Also the description of the pace - as you say people want to stretch themselves.  It's not exclusive but from what I gather it is supposed to be somewhere between the social and a training ride.